clients/drcachesim/tools/view.h

```diff
@@ -87,7 +87,7 @@ class view_t : public analysis_tool_t {
     inline void
     print_header()
     {
-        std::cerr << std::setw(9) << "Output format:\n<record#> <instr#>"
+        std::cerr << std::setw(12) << "Output format:\n<record#> <instr#>"
```

Comment:
Looks like the record# and instr# headers no longer align with their columns below: please fix.